src/cronjobs/userProfileJobs.js

```diff
@@ -1,26 +1,25 @@
-const { CronJob } = require('cron');
-const moment = require('moment-timezone');
-
-const userhelper = require('../helpers/userHelper')();
+const { CronJob } = require("cron");
+const moment = require("moment-timezone");
```

Comment:
1. The _moment-timezone_ library is deprecated. Consider using the `luxon` or ` dayjs` libraries instead.

2. We can directly use a task scheduler also like _bull_ or _agenda_ instead of relying on a cron job

Reply:
Gotcha that's a bit outside of this PR so let's add it to the bug sheet!